### ISSUE TITLE
Kubectl command failed:

### ISSUE THREAD

**Issue by ajaiswal-conga** [vscode-kubernetes-tools/vscode-kubernetes-tools#1366]
Type: <b>Bug</b>

I am trying to reload the cluster and getting this error "Kubectl command failed:". I have my own cofig file in ~/.kube/config and unable to see cluster list in vscode.

Extension version: 1.3.18
VS Code version: Code 1.94.2 (384ff7382de624fb94dbaf6da11977bba1ecd427, 2024-10-09T16:08:44.566Z)
OS version: Darwin arm64 24.0.0
Modes:

<details>
<summary>System Info</summary>

|Item|Value|
|---|---|
|CPUs|Apple M1 Pro (8 x 2400)|
|GPU Status|2d_canvas: enabled<br>canvas_oop_rasterization: enabled_on<br>direct_rendering_display_compositor: disabled_off_ok<br>gpu_compositing: enabled<br>multiple_raster_threads: enabled_on<br>opengl: enabled_on<br>rasterization: enabled<br>raw_draw: disabled_off_ok<br>skia_graphite: disabled_off<br>video_decode: enabled<br>video_encode: enabled<br>webgl: enabled<br>webgl2: enabled<br>webgpu: enabled<br>webnn: disabled_off|
|Load (avg)|5, 4, 4|
|Memory (System)|32.00GB (0.39GB free)|
|Process Argv|--crash-reporter-id 19583eb1-e1eb-4b39-bf69-d32ca09e1ac8|
|Screen Reader|no|
|VM|0%|
</details><details>
<summary>A/B Experiments</summary>

```
vsliv368cf:30146710
vspor879:30202332
vspor708:30202333
vspor363:30204092
vscod805:30301674
binariesv615:30325510
vsaa593cf:30376535
py29gd2263:31024239
cp13037t-nofilt:31155934
c4g48928:30535728
azure-dev_surveyone:30548225
a9j8j154:30646983
962ge761:30959799
pythongtdpath:30769146
pythonnoceb:30805159
asynctok:30898717
pythonmypyd1:30879173
h48ei257:31000450
pythontbext0:30879054
accentitlementst:30995554
cppperfnew:31000557
dsvsc020:30976470
pythonait:31006305
dsvsc021:30996838
bdiig495:31013172
a69g1124:31058053
dvdeprecation:31068756
dwnewjupyter:31046869
newcmakeconfigv2:31071590
impr_priority:31102340
nativerepl1:31139838
refactort:31108082
pythonrstrctxt:31112756
wkspc-onlycs-t:31132770
wkspc-ranged-t:31151552
cf971741:31144450
defaultse:31146405
iacca2:31156134
notype1:31157159
5fd0e150:31155592
iconenabled:31158251

```

</details>

**Comment by ajaiswal-conga** [comment]
This is resolved

**Comment by sw360cab** [comment]
I am encountering the issue.
BUT In my case It is not a regression, rather a matter of VS Code itself.

Indeed I moved around some folders related to projects that are in the current workspace.
By doing that I forgot to reimport some moved projects into the workspace.

I found out that this broke completely the plugin. 
Reimporting the given projects from the correct folders automatically solved the issue.
